bucket/deltachat.json

```diff
@@ -0,0 +1,36 @@
+{
+    "version": "1.3.4",
+    "description": "Email-based secure instant messaging",
+    "homepage": "https://delta.chat",
```

Comment:
```suggestion
    "homepage": "https://delta.chat/en/",
```

![image](https://user-images.githubusercontent.com/13260377/83325785-681caa00-a26f-11ea-8afa-6be68eaa4e19.png)

Reply:
I guess it is redirected to different url based on the location. Users can see the page in their languages.

Reply:
Use english